### PR TITLE
Lower fees as voted plz

### PR DESCRIPTION
**Pull request by Arrowana**
Special fee tier for stable needs more thinking so this should come later.

https://dao.projectserum.com/#/proposal/7GxS9nEc4ZiTjgSpTxg7JnsJXfyayoJocSdzAj5PmWR8?programId=AVoAYTs36yB5izAaBkxRG67wL1AMwG3vo41hKtUSb8is

### CODE REVIEW

**Comment by siong1987** [comment]
+1 for this. this requires no changes for anyone, maybe the communication to let people know about this. and, we can have the new fees immediately for everyone.

File: dex/src/fees.rs [hunk 1]
```diff
@@ -56,13 +56,8 @@ impl U64F64 {
 }
 
 #[inline(always)]
-const fn fee_bps(bps: u64) -> U64F64 {
-    U64F64(((bps as u128) << 64) / 10_000)
-}
-
-#[inline(always)]
-const fn rebate_bps(bps: u64) -> U64F64 {
```
Comment (jhlx) [inline dex/src/fees.rs:64]: Maybe we keep this as rebate_tenth_of_bps for use below.

File: dex/src/fees.rs [hunk 2]
```diff
@@ -81,25 +76,20 @@ impl FeeTier {
...
     #[inline]
-    pub fn maker_rebate(self, pc_qty: u64) -> u64 {
-        use FeeTier::*;
-        let rate: U64F64 = match self {
-            MSRM => rebate_bps(5),
-            Base | SRM2 | SRM3 | SRM4 | SRM5 | SRM6 => rebate_bps(3),
-        };
-        rate.mul_u64(pc_qty).floor()
+    pub fn maker_rebate(self, _pc_qty: u64) -> u64 {
+        0
```
Comment (jhlx) [inline dex/src/fees.rs:80]: Maybe we use rebate_tenth_of_bps(0).mul_u64(pc_qty).floor(). Not sure why we used to add a 2^-64 rebate to every tier before, but can keep it just for consistency.

**Comment by jhlx** [comment]
Looks great, thank you so much!

**Pull request merged**